### PR TITLE
@craigspaeth - Schedule page

### PR DESCRIPTION
**Pull request by broskoski**
Uses mock data for now, but implements the functionality to close https://github.com/artsy/monolith/issues/10 . Shows a schedule of events for any given day, auto scrolls through them at a given interval and fades to an alert screen when the current time is within range of any alert set.

monolith-demo

### CODE REVIEW

File: app/collections/events.coffee [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+module.exports = class Events extends Collection
+  debugMode: true
+  model: Event
+
+  url: -> "#{env.EUROPA_ENDPOINT}/api/events"
+
+  currentEvents: ->
+    unless @debugMode
+      return @filter (model) ->
+        moment(model.get('start_time')).isSame new Date, 'day'
```
Comment (craigspaeth) [inline app/collections/events.coffee]: <3 moment.js

File: app/templates/alert.eco [hunk 2]
```diff
@@ -0,0 +1,22 @@
+<div class="events__corner-header corner-header">
+  Starting at <%= @item.formatDateTime('start_time', 'h:mma') %>
```
Comment (craigspaeth) [inline app/templates/alert.eco]: Look at that eco action!

**Comment by craigspaeth** [comment]
Looking real slick! Monolith has some very clean code. Neat to see a brunch app using the Ezel tools (benv/artsy-xapp-middleware/etc.) separately. The tests are likely failing from some global-ey state being shared across them causing the order that they're run to affect things. Sometimes the magic bullet is just `benv.teardown(false)`.

**Comment by craigspaeth** [comment]
:+1: merge when :green_heart:

File: app/templates/alert.eco [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+</div>
+
+<div class="events__details">
+  <div class="events__details__inner">
+    <div class="events__details__item">
+      <div class="events__details__item__time">
+        <%= "#{@item.formatDateTime('start_time', 'h:mm')}-#{@item.formatDateTime('end_time', 'h:mma')}" %>
```
Comment (dzucconi) [inline app/templates/alert.eco]: http://en.wikipedia.org/wiki/Dash#Ranges_of_values

**Pull request merged**